docs/overview.rst

```diff
@@ -37,5 +37,19 @@ YAML Configuration
 .. literalinclude:: ../yaml/minimal.yaml
    :language: yaml
 
+*Example:* a parallel batch job for adiabatic flame calculations with parameter variations
```

Comment:
Convert this to an actual heading, i.e.
```
Run Batch Job
*************
```

Further, the placement of the earlier headings appears to be non-ideal, as they follow the description.